src/markdown-pages/instant-observability/index.mdx

```diff
@@ -0,0 +1,18 @@
+---
+path: '/instant-observability'
+title: 'Instant observability'
+template: 'OverviewTemplate'
+description: ''
+---
+
+<Intro>
+
+New Relic have been providing different solutions for you to kick start your observability journey. 
+But wouldn't it be cool if you wouldn't have to manually set up the observability building block? Instead you could just click, install and get value from your data.
+
+New Relic I/O equips you with integrations, dashboards, and other observability building blocks to get value from your data faster. We have a growing number of quickstarts that you can use for instant observability.
+Didn't find what you're looking for? You're always welcome to create a quickstart and contribute it to help others instantly monitor and observe their services.
+
+<Video id="jN3Bk473R4w" type="youtube" />
```

Comment:
the official marketing video will not be releases until 10.13, so you'll want t use static image here until it's available.

https://github.com/newrelic/developer-website/issues/1708

Reply:
@jpvajda can you please point to the static image that should be used?

Reply:
We don't have one available, so you can either 

1. Ask marketing for one.  they did provide us a banner element that could be used. see the I/O main page
2. Use some generic image 
3. leave terraform up for now

i'll follow up in #quickstarts-internal and cc you, but I'd like for you to coordinate with marketing if you can so you can run point on it.

Reply:
Note this image can be used on the home page of developer. so that doesn't seem to be in this PR.

Reply:
I'll make one for that change @mehreentahir16